### PR TITLE
implementing the option to save fuel state

### LINKED ISSUE #503: Fuel quantity state could be saved between sessions
Fuel quantity state could be saved between sessions if the user activates it via a toggle in the aircraft menu (default = off, to avoid creating problems for people who "just want to fly").

### PR DESCRIPTION
**Pull request by gilbertohasnofb**
Closes #503

### CODE REVIEW

File: Nasal/c172p.nas [hunk 1]
```diff
@@ -30,6 +30,16 @@ controls.applyParkingBrake = func (v) {
 };
 
 ##########################################
+# Fuel Save State
+##########################################
+var fuel_save_state = func {
+    if (!getprop("/consumables/fuel/save-fuel-state")) {    
+        setprop("/consumables/fuel/tank[0]/level-gal_us",20);
```
Comment (onox) [inline Nasal/c172p.nas]: Small remark: please separate parameters by one space.

**Comment by onox** [comment]
Code looks OK. Have one small remark. Did not test PR in FG.

**Comment by gilbertohasnofb** [comment]
Thanks for the remark, I fixed the spacing. I tested it myself and it is working to me, but maybe we want to double check before merging. But take your time with that, please.

**Comment by gilbertohasnofb** [comment]
Thanks!

**Pull request merged**